<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TradeNameDialog</class>
 <widget class="QDialog" name="TradeNameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Trade Name</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="labelNamespace">
       <property name="text">
        <string notr="true">Domain</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string notr="true">example.bit</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     
     <item row="4" column="0">
      <widget class="QLabel" name="amountLabel">
       <property name="text">
        <string>Requested &amp;amount:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>amountEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <spacer name="verticalSpacerAmountData">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="dataLabel">
       <property name="text">
        <string>&amp;Data:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>dataEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="dataEditHorizontalLayout">
       <item>
        <widget class="QLineEdit" name="dataEdit">
         <property name="toolTip">
          <string>Enter JSON string that will be associated with the name</string>
         </property>
         <property name="maxLength">
          <number>520</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDNSEditor">
         <property name="text">
          <string>DNS Editor&#8230;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="dataHintLabel">
       <property name="text">
        <string>JSON string, e.g. {&amp;quot;ip&amp;quot;: [&amp;quot;1.2.3.4&amp;quot;, &amp;quot;1.2.3.5&amp;quot;]}&lt;br&gt;See &lt;a href=&quot;https://github.com/namecoin/proposals&quot;&gt;JSON value specifications&lt;/a&gt;.</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <spacer name="verticalSpacerDataTransferTo">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelTransferTo">
       <property name="text">
        <string>&amp;Transfer to:</string>
       </property>
       <property name="buddy">
        <cstring>transferTo</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <layout class="QHBoxLayout" name="transferToLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPayToEdit" name="transferTo">
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="labelTransferToHint">
       <property name="text">
        <string>(Leave empty if not transferring.)</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <spacer name="verticalSpacerTransferToOffer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="labelOffer">
       <property name="text">
        <string>&amp;Buy Offer to accept:</string>
       </property>
       <property name="buddy">
        <cstring>inputOffer</cstring>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <layout class="QHBoxLayout" name="offerLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="inputOffer">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <!--
     <item row="13" column="1">
      <widget class="QLabel" name="labelOfferHint">
       <property name="text">
        <string>(Leave empty if creating a new offer.)</string>
       </property>
      </widget>
     </item>
     -->
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_amountEdit">
       <item>
        <!-- TODO: class BTCAmountEdit -->
        <widget class="QLineEdit" name="amountEdit">
         <property name="toolTip">
          <string>Amount in NMC</string>
         </property>
        </widget>
       </item>
       <item>
        <!-- TODO: class AmountEdit -->
        <widget class="QLineEdit" name="fiatAmountEdit">
         <property name="toolTip">
          <string>Amount in fiat</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelSubmitSellHint">
     <property name="text">
         <string notr="true">If you leave the Buy Offer field blank, you are committing to sell this name to anyone who pays the specified price.&lt;br&gt;&lt;br&gt;If you want control over whom you sell this name to (or if you already possess a Buy Offer), paste a specific Buy Offer to accept (the trade will take approximately 10 minutes to 2 hours).</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelSubmitBuyHint">
     <property name="text">
         <string notr="true">If you leave the Sell Offer field blank, you are committing to buy this name from anyone who owns it and accepts the specified price.&lt;br&gt;&lt;br&gt;If you already possess a Sell Offer, paste it to accept (the trade will take approximately 10 minutes to 2 hours).</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="outputOffer">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="plainText">
         <string>0123456789abcdef</string>
     </property>
     <property name="readOnly">
         <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelOutputSellHint">
     <property name="text">
         <string notr="true">Publish the above Sell Offer wherever a potential buyer might see it.  Once you've published it, click OK to freeze this name so that the Sell Offer stays valid.  You can change your mind and revoke your Sell Offer by unfreezing this name and then updating or renewing it.  If you want to update or renew this name while waiting for a buyer, you will need to unfreeze it first and then re-create and re-publish the Sell Offer after issuing the update.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelOutputBuyHint">
     <property name="text">
         <string notr="true">Publish the above Buy Offer wherever the current name owner might see it.  Once you've published it, click OK to freeze the coins that you used for this Buy Offer so that the Buy Offer stays valid.  You can change your mind and revoke your Buy Offer by unfreezing the coins used in this Buy Offer and then spending them.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPayToEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>qpaytoedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TradeNameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
